static/app/components/events/interfaces/crashContent/stackTrace/rawContent.spec.tsx

```diff
@@ -130,20 +130,78 @@ describe('RawStacktraceContent', () => {
     it('renders javascript example', () => {
       expect(displayRawContent(data, 'javascript', exception)).toBe(
         `Error: an error occurred
-\tat doThing3 (example.application:12:24)
-\tat ? (src/application.code:1:6)
-\tat doThing1 (src/application.code:5:9)
-\tat main (src/application.code:1:14)`
+    at doThing3 (example.application:12:24)
+    at ? (src/application.code:1:6)
+    at doThing1 (src/application.code:5:9)
+    at main (src/application.code:1:14)`
+      );
+    });
+
+    it('renders javascript example - rawTrace, newestFirst', () => {
+      expect(
+        displayRawContent(data, 'javascript', exception, false, true, true, true)
+      ).toBe(
+        `Error: an error occurred
+    at doThing3 (example.application:12:24)
+    at ? (src/application.code:1:6)
+    at doThing1 (src/application.code:5:9)
+    at main (src/application.code:1:14)`
+      );
+    });
+
+    it('renders javascript example - !rawTrace, newestFirst', () => {
+      expect(
+        displayRawContent(data, 'javascript', exception, false, true, false, true)
+      ).toBe(
+        `Error: an error occurred
+    at doThing3 (example.application:12:24)
+    at ? (src/application.code:1:6)
+    at doThing1 (src/application.code:5:9)
+    at main (src/application.code:1:14)`
+      );
+    });
+
+    it('renders javascript example - rawTrace, !newestFirst', () => {
+      expect(
+        displayRawContent(data, 'javascript', exception, false, true, true, false)
+      ).toBe(
+        `Error: an error occurred
+    at doThing3 (example.application:12:24)
+    at ? (src/application.code:1:6)
+    at doThing1 (src/application.code:5:9)
+    at main (src/application.code:1:14)`
+      );
+    });
+
+    it('renders javascript example - !rawTrace, !newestFirst', () => {
+      expect(
+        displayRawContent(data, 'javascript', exception, false, true, false, false)
+      ).toBe(
+        `    at main (src/application.code:1:14)
+    at doThing1 (src/application.code:5:9)
+    at ? (src/application.code:1:6)
+    at doThing3 (example.application:12:24)
+Error: an error occurred`
```

Comment:
Are we okay with this being the format for `newestFirst = false` stack traces (the exception summary line being at the bottom)? Do we still want that at the top?

I think it makes more sense at the bottom, but it might be a little strange to show
`    at main (src/application.code:1:14)`
as the first line. Maybe we could do something similar to Python and add an additional line at the top specifying that the most recent call is last (since this diverges from default behavior)?